### ISSUE TITLE
Middleman 4.0 support?

### ISSUE THREAD

**Issue by tsigo** [ngs/middleman-ogp#4]
Hi @ngs, just curious if you have plans (or might be willing) to update this for Middleman 4.0?

**Comment by pachirel** [comment]
@ngs @tsigo I'm working to support middleman-ogp on middleman v4.0.

[My code](https://github.com/pachirel/middleman-ogp/tree/middleman-v4) works fine for me, but spec is failing. 😢

```
% bundle exec rake test
/Users/pachirel/.rbenv/versions/2.2.2/bin/ruby -I/Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib:/Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-support-3.4.1/lib /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
/Users/pachirel/src/github.com/pachirel/middleman-ogp/lib/middleman-ogp/extension.rb:62:in `<module:Helper>': uninitialized constant Middleman::OGP::Helper::Padrino (NameError)
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/lib/middleman-ogp/extension.rb:61:in `<module:OGP>'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/lib/middleman-ogp/extension.rb:2:in `<module:Middleman>'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/lib/middleman-ogp/extension.rb:1:in `<top (required)>'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/spec/spec_helper.rb:5:in `require'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/spec/spec_helper.rb:5:in `<top (required)>'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/spec/helper_spec.rb:1:in `require'
	from /Users/pachirel/src/github.com/pachirel/middleman-ogp/spec/helper_spec.rb:1:in `<top (required)>'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/configuration.rb:1361:in `load'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/configuration.rb:1361:in `block in load_spec_files'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/configuration.rb:1359:in `each'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/configuration.rb:1359:in `load_spec_files'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/runner.rb:106:in `setup'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/runner.rb:92:in `run'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/runner.rb:78:in `run'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib/rspec/core/runner.rb:45:in `invoke'
	from /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/exe/rspec:4:in `<main>'
/Users/pachirel/.rbenv/versions/2.2.2/bin/ruby -I/Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/lib:/Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-support-3.4.1/lib /Users/pachirel/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rspec-core-3.4.4/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb failed
```

I don't understand what's wrong with my code and setup.
Any support are welcome 😃

**Comment by ngs** [comment]
@pachirel I just merged a branch using your changes #5 Thanks for your contribution!
